resources/views/tasks.blade.php

```diff
@@ -33,49 +25,38 @@
                         </div>
                     </form>
                 </div>
-
-                {{-- TODO: Current Tasks --}}
-                @if(count($tasks) > 0)
+                @if(isset($tasks)&&count($tasks) > 0)
```

Comment:
```suggestion
                @if(isset($tasks) && count($tasks) > 0)
```
論理演算子の前後は半角スペース開けるようにしてください。

Reply:
論理演算子の前後に半角スペースを追加致しました。
ご確認よろしくお願い致します。
app/Http/Controllers/TaskController.php